src/components/endpointCreation/NewCaptureModal.tsx

```diff
@@ -163,7 +165,9 @@ function NewCaptureModal(
         setNewCaptureDetailsFormErrors(errors);
 
         if (data.sourceType) {
-            getSourceDetails(data.sourceType);
+            if (data.sourceType !== 'custom') {
+                getSourceDetails(data.sourceType);
+            }
```

Comment:
Wanting to allow custom image paths. So when they choose "custom" we don't want to fetch anything yet.